src/styles/components/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  transition-duration: 0.3s;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    position: relative;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    position: relative;
+    padding: 23px 0;
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    &:hover {
+      color: $color-active;
+    }
+  }
+}
+
+.is-active {
+  color: $color-active;
+}
```

Comment:
The `.is-active` class should be nested within `.nav__link` using the `&` operator to maintain the BEM structure and SCSS nesting conventions. For example, it should be inside `.nav__link` as `&.is-active { color: $color-active; }`.